docs/generators.md

```diff
@@ -96,6 +96,8 @@ scope and thus capturing references is dangerous. If you need to use
 variables inside the generator expression, make sure you thought through
 the lifetime implications and use `GENERATE_COPY` or `GENERATE_REF`.**
 
+**Note**: `GENERATE_COPY` and `GENERATE_REF` were introduced in Catch 2.7.1.
```

Comment:
I think this would look better if it used the same convention as the rest of these...

> `GENERATE_COPY` and `GENERATE_REF` were introduced in Catch 2.7.1.

Reply:
Thanks for the feedback! Done in 3d91219 ...

Reply:
I later realised that you meant to use the Quote style - with `> ` prefix on this line, which is much better,

I've now done that for all in 3f04c2f.